zcash_client_backend/src/data_api.rs

```diff
@@ -909,6 +909,20 @@ pub trait WalletRead {
     ) -> Result<HashMap<TransparentAddress, NonNegativeAmount>, Self::Error> {
         Ok(HashMap::new())
     }
+
+    /// Returns the set of reserved ephemeral addresses controlled by this wallet.
+    ///
+    /// The set contains all ephemeral transparent receivers that are known to have
+    /// been derived under this account. Wallets should scan the chain for UTXOs sent to
+    /// these receivers.
+    #[cfg(feature = "transparent-inputs")]
+    fn get_reserved_ephemeral_addresses(
```

Comment:
I don't think that "reserved" needs to be a part of this title; an ordinary transparent address can also be generated but then not be provided to a counterparty.

Reply:
"Reserved" specifically means the subset of ephemeral addresses that we have generated so far.